src/detect.h

```diff
@@ -244,6 +244,7 @@ typedef struct DetectPort_ {
 
 #define SIG_FLAG_DSIZE                  BIT_U32(5)  /**< signature has a dsize setting */
 #define SIG_FLAG_APPLAYER               BIT_U32(6) /**< signature applies to app layer instead of packets */
+#define SIG_FLAG_BOTHDIR                BIT_U32(7) /**< signature needs both directions to match */
 
 // vacancy
```

Comment:
remove now that bit 7 is used

Reply:
But still vacancy for BIT_U32(8) ;-)

Reply:
fair. Github confused me as it shows BIT_U32(8) but from a different set of flags